platforms/core-runtime/launcher/src/test/groovy/org/gradle/launcher/cli/converter/StartParameterConverterTest.groovy

```diff
@@ -91,20 +91,20 @@ class StartParameterConverterTest extends Specification {
     def "can provide start parameter option as command-line option"() {
         expect:
         def parameter = convert("--configuration-cache")
-        parameter.configurationCache.get()
+        parameter.getConfigurationCache().get()
```

Comment:
What changed here?

Reply:
`StartParameterInternal` has `isConfigurationCache()` and `getConfigurationCache()`, the first one is selected by 4.x and the second is selected by 3.x due to the `boolean` property changes.